.nvmrc

```diff
@@ -1 +1 @@
-12.16.1
+16.1.0
```

Comment:
We might need to add this to the doc as well.

Reply:
At the moment we aren't using nvm to manage node.js.  It's scattered across the platforms.
Let's take this as a subsequent change to ensure we use a consistent version of the node everywhere. 

https://github.com/chatwoot/docs/issues/66